CMakeLists.txt

```diff
@@ -84,6 +84,11 @@ if(WITH_OPENMP AND USE_BUNDLED_LLVMOPENMP)
 else()
   set(ALLOW_DOWNLOADING_LLVMOPENMP OFF CACHE BOOL "If LLVM OpenMP runtime library src tree is not found in location specified by LLVMOPENMP_PATH, do fetch the archive from internet" FORCE)
 endif()
+if(WITH_OPENMP AND NOT USE_BUNDLED_LLVMOPENMP AND NOT ALLOW_DOWNLOADING_LLVMOPENMP)
+  find_package(OpenMP REQUIRED)
```

Comment:
This is all already done in https://github.com/darktable-org/rawspeed/blob/344f3b94f37e0bc9b82f6b71047d489bd598311a/cmake/src-dependencies.cmake#L19-L102